x-pack/plugins/uptime/public/lib/index.ts

```diff
@@ -5,4 +5,4 @@
  */
 
 export { MountWithReduxProvider } from './helper';
-export * from './helper/helper_with_router';
+export * from './helper/enzyme_helpers';
```

Comment:
Exported the enzyme helpers from here in order to maintain compatibility with the existing tests without having to change the imports.